### DISCUSSION TITLE
Setup for local plugin development

### DISCUSSION THREAD

**Discussion by mrjones2014** [wbthomason/packer.nvim#1168] [Q&A]
When I'm working on a plugin locally, I put the local path into packer like the README.md suggests.

I also use [neodev.nvim](https://github.com/folke/neodev.nvim). This setup, however, results in incorrect "duplicate field" diagnostics all over the place. I think this is because the local plugin is also symlinked at `~/.local/share/nvim/site/pack/packer/opt/[plugin name]`, so the diagnostics are getting the "duplicates" from my runtime path, while the current repository directory should be the real source of truth.

![image](https://user-images.githubusercontent.com/8648891/207320500-a6e30bcd-8d28-411f-87f2-5739f0513e59.png)

Is there a way to get the diagnostics correctly realize that `~/.local/share/nvim/site/pack/packer/opt/[plugin name]` and `~/git/github/[repo name]` are actually the same directory and not produce incorrect "duplicate field" diagnostics?

**Comment by SiddidMandi** [comment]
https://github.com/wbthomason/packer.nvim/discussions/309

this discussion thread might help u

**Reply by mrjones2014** [reply]
I've since switched to `lazy.nvim` which just doesn't have this issue since it loads development plugins from the existing directory.
